tools/clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -7,6 +7,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "dxc/Support/Global.h"
```

Comment:
I believe minimal include is actually this header:
```
#include "dxc/Support/exception.h"  // HLSL Change
```

Reply:
Also, this should probably be included at the bottom of the list with the other dxc includes.

Reply:
Also, this should probably be included at the bottom of the list with the other dxc includes.

Reply:
There are several build errors when just including exception, that are resolved by including global:
Error	C3646	'hr': unknown override specifier	clangCodeGen	D:\DirectXShaderCompiler\include\dxc\Support\exception.h	28	
Error	C4430	missing type specifier - int assumed. Note: C++ does not support default-int	clangCodeGen	D:\DirectXShaderCompiler\include\dxc\Support\exception.h	28	
Error	C2061	syntax error: identifier 'HRESULT'	clangCodeGen	D:\DirectXShaderCompiler\include\dxc\Support\exception.h	31